deepfence_server/controls/agent.go

```diff
@@ -14,22 +14,37 @@ import (
 )
 
 var (
+	MaxStops    = 5
```

Comment:
This `MaxStops` is a bit misleading because it only applies for 1 iteration: Let's imagine we cancel 30 scans, we will do: 5 cancel, next 30 seconds 5 more, next 30 seconds 5 more, without checking if the previous cancel were completed.

Now I think in practice we should be fine because we expect cancels to finish quickly and the request should not use extra resources to complete.. I would rather remove that `MaxStops` entirely, just to simplify our logic and not mislead us in the future

Reply:
I believe we would need to put in a limit to how many we can process in a one go.   
Maybe we can raise this to higher number (maybe 50)